File: Java/Program/Mouse.java
public class Mouse extends Periferal {
    private String tipe;
    private int dpi;
    private String tipeKoneksi;

    public Mouse(String merk, String nama, String koneksi, String tipe, int dpi, String tipeKoneksi) {
        super(merk, nama, koneksi);
        this.tipe = tipe;
        this.dpi = dpi;
        this.tipeKoneksi = tipeKoneksi;
    }

    public void setTipe(String tipe) {
        this.tipe = tipe;
    }

    public String getTipe() {
        return this.tipe;
    }

    public void setDpi(int dpi) {
        this.dpi = dpi;
    }

    public int getDpi() {
        return this.dpi;
    }

    public void setTipeKoneksi(String tipeKoneksi) {
        this.tipeKoneksi = tipeKoneksi;
    }

    public String getTipeKoneksi() {
        return this.tipeKoneksi;
    }
}

File: Python/Program/Mouse.py
from Periferal import Periferal

class Mouse(Periferal):
    def __init__(self, merk, nama, koneksi, tipe, dpi, tipe_koneksi):
        super().__init__(merk, nama, koneksi)
        self.tipe = tipe
        self.dpi = dpi
        self.tipe_koneksi = tipe_koneksi
        
    def setTipe(self, tipe):
        self.tipe = tipe
        
    def getTipe(self):
        return self.tipe

    def setDpi(self, dpi):
        self.dpi = dpi

    def getDpi(self):
        return self.dpi

    def setTipeKoneksi(self, tipe_koneksi):
        self.tipe_koneksi = tipe_koneksi

    def getTipeKoneksi(self):
        return self.tipe_koneksi